### PR TITLE
fix(autoware_radar_object_clustering): fix funcArgNamesDifferent

### PR DESCRIPTION
**Pull request by kobayu858**
## Description
This is a fix based on cppcheck funcArgNamesDifferent warnings

```
perception/radar_object_clustering/src/radar_object_clustering_node.cpp:94:81: style: inconclusive: Function 'onObjects' argument 1 names different: declaration 'msg' definition 'objects_data_'. [funcArgNamesDifferent]
void RadarObjectClusteringNode::onObjects(const DetectedObjects::ConstSharedPtr objects_data_)
                                                                                ^
```

## Related links

**Parent Issue:**

- Link

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Comment by veqcc** [comment]
Same comment https://github.com/autowarefoundation/autoware.universe/pull/8013#issuecomment-2225300314

**Comment by kobayu858** [comment]
@scepter914 @miursh @technolojin @YoshiRi 

Please confirm the PR. Thank you very much.

**Comment by yukkysaito** [comment]
It appears that objects_data_ is not used in this change. objects_data_ is likely a typo. It would be better to align the code with msg instead of objects_data_.

File: perception/autoware_radar_object_clustering/src/radar_object_clustering_node.hpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ class RadarObjectClusteringNode : public rclcpp::Node
   rclcpp::Subscription<DetectedObjects>::SharedPtr sub_objects_{};
 
   // Callback
-  void onObjects(const DetectedObjects::ConstSharedPtr msg);
+  void onObjects(const DetectedObjects::ConstSharedPtr objects_data_);
```
Comment (YoshiRi) [inline perception/autoware_radar_object_clustering/src/radar_object_clustering_node.hpp:55]: I think it's better to keep the original declaration and rename the .cpp function instead.

**Review by YoshiRi — CHANGES_REQUESTED** [review]

File: perception/autoware_radar_object_clustering/src/radar_object_clustering_node.hpp (hunk 1 above)
Comment (kobayu858) [reply perception/autoware_radar_object_clustering/src/radar_object_clustering_node.hpp]: Thanks for the advice.
I removed objects_data_ and unified it with msg.

This response is the same as [PR#8013](https://github.com/autowarefoundation/autoware.universe/pull/8013).

**Review by YoshiRi — APPROVED** [review]
LGTM

**Pull request merged**